CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 Change Log
 ==========
+Version 0.3.0 *(2019-02-04)*
+----------------------------
+
+* Prefill attachments  
+
+Bug Fixes:
+
+* Ensured all places the Beacon Config is accessed it has the overrides applied 
+* Crash when launching BeaconActivity with Context rather than Activity 
```

Comment:
I'd prefer we just say 'Improved stability by resolving several crashes' unless these were customer reported - if customer reported is this how they raised the issue?